src/Client/DefaultRegistrationService.php

```diff
@@ -57,6 +57,10 @@ public function register(iterable $strategyHandlers): bool
                 'strategies' => array_map(fn (StrategyHandler $strategyHandler): string => $strategyHandler->getStrategyName(), $strategyHandlers),
                 'started' => (new DateTimeImmutable())->format('c'),
                 'interval' => $this->configuration->getMetricsInterval(),
+                'platformName' => PHP_SAPI,
+                'platformVersion' => phpversion(),
```

Comment:
```suggestion
                'platformVersion' => PHP_VERSION,
```

It doesn't really make any difference, but constant is faster than function call. Perhaps a pointless optimization, so feel free to ignore.

Reply:
I'm a hopeless nerd for micro optimizations, merged, thanks!